### ISSUE TITLE
Do not commit reset Chart.yaml and values.yaml file when updating crd docs

### ISSUE THREAD

**Issue by olevski** [SwissDataScienceCenter/amalthea#98]
It was not very obvious to me when I was initially reviewing the code. But it is weird that the release that is tagged in github with the release tag actually contains a chart that does not have that tag. Then in a subsequent release we rely on the automated crd docs generation workflow to get the chart tag that matches the release tag.

I think it is better to make the workflow that updates the crd docs not change the values.yaml and Chart.yaml file. Or if this cannot be avoided then we can just not commit these files at all when the crd docs and manifests are committed. This way the `0.1.0` release will indeed contain the `0.1.0` chart.

**Comment by ableuler** [comment]
> But it is weird that the release that is tagged in github with the release tag actually contains a chart that does not have that tag.

That is true, I think it would be good have the correct chart version in the tagged version of the repo. But I don't think that this conflicts with the current function of our github workflow. If we bump the `resetVersion` in a release-prep PR, say from 0.1.0 to 0.1.1, the `generate crd docs` workflow will run `chartpress --reset` thereby bumping the chart version itself from 0.1.0 to 0.1.1 and open another PR with the results. So we can just wait for this auto-generated PR to be merged too before tagging the release, or we skip the auto-pr by just running `chartpress --reset` locally and including the result in the release-prep PR.

**Comment by ableuler** [comment]
I actually agree that the current setup is not yet perfect. I think the solution is to use more 🤖s. I'll give it a shot, but after the 1.1.0 release is out.
